lib/src/generateRunner.cc

```diff
@@ -2463,6 +2463,14 @@ void genRunnerGPU(const NNmodel &model, //!< Model description
             os << "push" << n.first << "SpikesToDevice(hostInitialisedOnly);" << std::endl;
         }
 
+        for(const auto &cs : model.getRemoteCurrentSources()) {
+            os << "push" << cs.first << "StateToDevice(hostInitialisedOnly);" << std::endl;
+        }
```

Comment:
Remote one shouldn't be pushed as they're sitting on another MPI node (in some theretical world where the MPI support works!)